File: BAEKJOON/Node/2000/2400/2476.js
const fs = require("fs")
const filepath = process.platform === "linux" ? "/dev/stdin" : "./input.txt"
const input = fs.readFileSync(filepath).toString().trim().split("\n")

const N = parseInt(input[0])
let answer = 0
for (let i = 1; i <= N; i++) {
  let [a, b, c] = input[i].split(" ").map(x => parseInt(x))
  let tmp = 0
  if (a === b && b === c) {
    tmp = 10000 + 1000 * a
  } else if (a === b) {
    tmp = 1000 + 100 * a
  } else if (b === c) {
    tmp = 1000 + 100 * b
  } else if (c === a) {
    tmp = 1000 + 100 * c
  } else {
    tmp = Math.max(a, b, c) * 100
  }
  if (answer < tmp) {
    answer = tmp
  }
}
console.log(answer)

File: BAEKJOON/Python/2000/2400/2476.py
"""
Title : 주사위 게임
Link : https://www.acmicpc.net/problem/2476
"""

from sys import stdin

input = stdin.readline


if __name__ == "__main__":
    N = int(input())
    answer = 0
    for _ in range(N):
        a, b, c = map(int, input().split())
        if a == b == c:
            tmp = 10000 + a * 1000
        elif a == b:
            tmp = 1000 + 100 * a
        elif b == c:
            tmp = 1000 + 100 * b
        elif c == a:
            tmp = 1000 + 100 * c
        else:
            tmp = max(a, b, c) * 100
        if answer < tmp:
            answer = tmp
    print(answer)
